### PR TITLE
Add missing EventSummary `note` prop documentation

### PR DESCRIPTION
**Pull request by gerardo-rodriguez**
## Overview

Not critical, but something I noticed along the way.

### CODE REVIEW

**Review by spaceninja — DISMISSED** [review]

**Comment by gerardo-rodriguez** [comment]
Thanks, @spaceninja. Looks like my deleting the changeset file dismissed your approval. Can you approve again so I can merge (the merge button is disabled otherwise)?

**Review by spaceninja — APPROVED** [review]

**Pull request merged**
